ui.apps/src/main/content/jcr_root/apps/aemtraining/components/content/newscard/newscard.html

```diff
@@ -8,6 +8,19 @@
     <div class="wrapper">
         <div class="box">
             <img src="${newscard.image}" alt="">
+            <div class="like_dislike" style="width:400px">
+                <form action="${request.requestURI}" class="form_dislike" style="float: left; width: 130px">
```

Comment:
I believe this is a draft, yeah? Because this form does virtually nothing as of now, just reloads the page as is.

Reply:
yeah it was not meant to be here).